modules/tinymce/src/core/main/ts/caret/TableCells.ts

```diff
@@ -73,8 +73,8 @@ const getTopValue = (rect: ClientRect) => rect.top;
 const isAbove = (corner: Corner, y: number) => corner.y < y;
 const isBelow = (corner: Corner, y: number) => corner.y > y;
 
-const getClosestCellAbove = Fun.curry(getClosestCell, getBottomValue, isAbove);
-const getClosestCellBelow = Fun.curry(getClosestCell, getTopValue, isBelow);
+const getClosestCellAbove = Fun.curry(getClosestCell, getBottomValue, isAbove) as (table: HTMLElement, x: number, y: number) => Optional<CellOrCaption>;
```

Comment:
Actually, if you look everywhere else this is already used in the caret code it uses the above style...

Reply:
Having said that though, you are right we normally use the other style. Will consider fixing all of them here, otherwise I'll log a JIRA to go through the entire code base to standardise them.